src/erlfmt.erl

```diff
@@ -69,6 +74,30 @@ format_file(FileName, Out) ->
         {error, Error} -> {error, Error}
     end.
 
+-spec format_range(
+    file:name_all(),
+    erlfmt_scan:location(),
+    erlfmt_scan:location()
+) ->
+    {ok, string(), [error_info()]} |
+    {error, error_info()} |
+    {options, [{erlfmt_scan:location(), erlfmt_scan:location()}]}.
```

Comment:
I wonder - if we expose this operation through CLI, what do you think should be the behaviour in case there are multiple options?

Reply:
Yeah I guess either there could be an interactive choice, or there is a default way to resolve these choice that can be altered with command line flags. I.e. the default is always choose the min range but you could choose to include a half baked form if it precedes or the range and same at the end.